sources/packages/backend/apps/db-migrations/src/sql/Student/Create-student-history-table.sql

```diff
@@ -0,0 +1,76 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.students_history (
+  history_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  history_operation VARCHAR(20) NOT NULL,
+  id INT,
+  contact_info JSONB,
+  user_id INT,
+  created_at TIMESTAMP WITHOUT TIME ZONE,
+  updated_at TIMESTAMP WITHOUT TIME ZONE,
```

Comment:
Created_at and updated_at are with timezone right?

Reply:
The schemas for the history tables reflect the current schema from the target table. If we change the target table schema we can then change the history table schema. Does it make sense?

Reply:
You were right, I got it now. I will update it.

Reply:
Fixed!